src/tools/wasm-merge.cpp

```diff
@@ -122,15 +122,15 @@ struct Mergeable {
     findImportsByBase(wasm, MEMORY_BASE, [&](Name name) {
       memoryBaseGlobals.insert(name);
     });
-    if (memoryBaseGlobals.size() == 0) {
-      Fatal() << "no memory base was imported";
-    }
+    //if (memoryBaseGlobals.size() == 0) {
```

Comment:
no need to leave the commented-out lines.

we will need tests for modules without these imports, in `test/merge/`